src/sentry/migrations/0309_fix_many_to_many_field.py

```diff
@@ -0,0 +1,149 @@
+                    ),
+                ),
+                migrations.AddField(
+                    model_name="authproviderdefaultteams",
+                    name="authprovider",
+                    field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to="sentry.AuthProvider"
+                    ),
+                ),
+                migrations.AddField(
+                    model_name="authproviderdefaultteams",
+                    name="team",
+                    field=sentry.db.models.fields.foreignkey.FlexibleForeignKey(
+                        on_delete=django.db.models.deletion.CASCADE, to="sentry.Team"
+                    ),
+                ),
+                migrations.AlterUniqueTogether(
+                    name="dashboardproject",
+                    unique_together={("project", "dashboard")},
+                ),
+                migrations.AlterUniqueTogether(
+                    name="authproviderdefaultteams",
+                    unique_together={("authprovider", "team")},
+                ),
+            ],
+        ),
+        migrations.RunSQL(
+            sql="ALTER TABLE sentry_authprovider_default_teams ALTER COLUMN id TYPE bigint",
+            hints={"tables": ["sentry_authprovider_default_teams"]},
+            reverse_sql="ALTER TABLE sentry_authprovider_default_teams ALTER COLUMN id TYPE int",
+        ),
+        migrations.RunSQL(
+            sql="ALTER TABLE sentry_authprovider_default_teams ALTER COLUMN authprovider_id TYPE bigint",
+            hints={"tables": ["sentry_authprovider_default_teams"]},
+            reverse_sql="ALTER TABLE sentry_authprovider_default_teams ALTER COLUMN authprovider_id TYPE int",
+        ),
+        migrations.RunSQL(
+            sql="ALTER TABLE sentry_authprovider_default_teams ALTER COLUMN team_id TYPE bigint",
+            hints={"tables": ["sentry_authprovider_default_teams"]},
+            reverse_sql="ALTER TABLE sentry_authprovider_default_teams ALTER COLUMN team_id TYPE int",
+        ),
+        migrations.RunSQL(
+            sql="ALTER TABLE sentry_dashboardproject ALTER COLUMN id TYPE bigint",
+            hints={"tables": ["sentry_dashboardproject"]},
+            reverse_sql="ALTER TABLE sentry_dashboardproject ALTER COLUMN id TYPE int",
+        ),
+        migrations.RunSQL(
+            sql="ALTER TABLE sentry_dashboardproject ALTER COLUMN id TYPE bigint",
+            hints={"tables": ["sentry_dashboardproject"]},
+            reverse_sql="ALTER TABLE sentry_dashboardproject ALTER COLUMN dashboard_id TYPE int",
+        ),
+        migrations.RunSQL(
+            sql="ALTER TABLE sentry_dashboardproject ALTER COLUMN id TYPE bigint",
+            hints={"tables": ["sentry_dashboardproject"]},
+            reverse_sql="ALTER TABLE sentry_dashboardproject ALTER COLUMN project_id TYPE int",
+        ),
+    ]
```

Comment:
I just noticed that you're not altering the fks here to be a bigint, you alter `id` several times